src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -1933,6 +1933,11 @@ elementFormDefault="qualified">
         </xs:annotation>
     </xs:element>
     <xs:element name="OSVersion" type="msb:StringPropertyType" substitutionGroup="msb:Property"/>
+    <xs:element name="BaseOutputPath" type="msb:StringPropertyType" substitutionGroup="msb:Property">
+        <xs:annotation>
+            <xs:documentation><!-- _locID_text="BaseOutputPath" _locComment="" -->Base path of output folder, to which configurations may be appended, with trailing slash</xs:documentation>
```

Comment:
Documentation I found says:
Specifies the base path for the output file. If it's set, MSBuild uses OutputPath = $(BaseOutputPath)\$(Configuration)\. Example syntax: <BaseOutputPath>c:\xyz\bin\</BaseOutputPath>

Unify the wording?

Reply:
I wondered if that is outdated as it doesn't include target frameworks or platforms in cases where they are relevant.

Reply:
Yeah—I have no problem with updating the documentation.

Reply:
I'm not confident enough in the area to doc it myself. My original motivation here was to get an entry in the XSD to support autocomplete in VS.

Reply:
There's intense layering confusion here--the MSBuild layer respects certain things, the SDK layer tweaks certain things. I like the comment in common.targets

https://github.com/dotnet/msbuild/blob/1ff019aaa7cc17f22990548bb19498dfbbdebaec/src/Tasks/Microsoft.Common.CurrentVersion.targets#L112-L114

I think mentioning the default value helps clarify.

```suggestion
            <xs:documentation><!-- _locID_text="BaseOutputPath" _locComment="" -->Base path of output folder, where all configuration-specific output folders will be created. Default value is bin\.</xs:documentation>
```